modules/dicehub/endpoints/endpoints.go

```diff
@@ -94,6 +96,12 @@ func WithPipelineTemplate(pipelineTemplate *template.PipelineTemplate) Option {
 	}
 }
 
+func WithReleaseRule(rule *release_rule.ReleaseRule) Option {
```

Comment:
Use proto to define new interfaces instead of endpoints

Reply:
Start a new pull request to fix it